.github/actions/telemetry_check_crwa/telemetry_check_crwa.mjs

```diff
@@ -0,0 +1,76 @@
+import { exec } from '@actions/exec'
+
+import http from "http"
+
+console.log(`Telemetry is being redirected to ${process.env.REDWOOD_REDIRECT_TELEMETRY}`)
+const port = parseInt(process.env.REDWOOD_REDIRECT_TELEMETRY.split(":")[2])
```

Comment:
If we don't use the bit before the colon, any reason we shouldn't just hardcode it? Just would like to avoid the duplication later when we call `server.listen`

Reply:
We can hard code the host and port for sure. My original plan was to extract both host and port from the redirect env. However, I got into a bother with the whole `localhost` vs `127.0.0.1`. 

I'll update to just hard code it all if that'd be the preference.

Reply:
> However, I got into a bother with the whole localhost vs 127.0.0.1

Ah yeah I've been there—not too much fun! And looking at the code again, the only potential issue with hardcoding is how will you know to set the `telemetryUrl` in `packages/telemetry/src/sendTelemetry.ts`? But if you've got that figured out go for it

Reply:
> Then the only things to keep in sync are the hard coded values in the CI action and the CI yaml itself

Sorry to belabor a small point but I think I got a little lost here—so we'd still be keeping hardcoded values in sync instead of trying to deduplicate them?

Reply:
No it's fine I'll make the CI script extract both host and port from the env var. Nothing to keep in sync that way.